### PR TITLE
adding minutes from 2018-10-01 meeting

### PR DESCRIPTION
**Pull request by mike-kaufman**
My notes were more sparse than usual, and I had technical difficulities during part of the call.  Please LMK if missed anything or got anything wrong.

### CODE REVIEW

**Comment by watson** [comment]
I unfortunately couldn't attend last night, but was really looking forward to viewing the recording today as the topics were super interesting. But I don't see it in the minutes?

**Comment by mike-kaufman** [comment]
@watson - really sorry about this.  I currently lack perms to create hangouts & recordings of the hangouts under the "node.js" identity, so I skipped the recording.  I opened issue #138 to address this.  We'll hopefully get it fixed up next time.  If you have any questions or want more color/clarity on anything in the notes, reach out, happy to do a call to fill you in.

File: wg-meetings/2018-01-10.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+## Attendees
+
+- @mike-kaufman
+- @qard(Stephen Belanger)
+- @AndreasMadsen
+- @danielkhan
+- @jankrems
+- @kjin
+- @ofrobots
+- @dominicdkramer
```
Comment (mhdawson) [inline wg-meetings/2018-01-10.md]: add @mhdawson

Comment (mike-kaufman) [reply wg-meetings/2018-01-10.md]: fixed

File: wg-meetings/2018-01-10.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+* \[async-hooks\] request for feedback [#129](https://github.com/nodejs/diagnostics/issues/129)
+  - Do we have enough feedback & is it useful? 
+  - [bluebird issue #1472](https://github.com/petkaantonov/bluebird/pull/1472) on perforance of async hooks is related to this.
+  - @danielkhan -  they're support down to node v4, and haven't had time to invest in a 8.x/9.x solution only - their monkey-patching still works
+  - @ofrobots - want to know if API is stable & useful
+  - **Actions**
+    - [ ] @mike-kaufman - summarize feedback in here & roll it into [async_hooks] stable API - tracking issue [#124](https://github.com/nodejs/diagnostics/issues/124) 
+
+* \[async_hooks\] stable API - tracking issue [#124](https://github.com/nodejs/diagnostics/issues/124)
+  -
```
Comment (mmarchini) [inline wg-meetings/2018-01-10.md]: Github interprets this trailing dash as a header:

image

Comment (mike-kaufman) [reply wg-meetings/2018-01-10.md]: fixed

**Pull request merged**